joss/paper.md

```diff
@@ -72,17 +72,17 @@ If further sampling from the posterior is necessary, the flow trained during a p
 The mathematical details of the method are explained in [@Gabrie2022; @Gabrie2021a].
 
 ***Use of accelerators***
-`flowMC` is built on top of JAX, that supports the use of GPU and TPU accelerators by default.
+`flowMC` is built on top of JAX, which supports the use of GPU and TPU accelerators by default.
```

Comment:
Not entirely sure about this one but 'that' doesn't read quite right to me here.